Makefile

```diff
@@ -48,10 +48,6 @@ version/version.go:
 	@echo "$(WHALE) $@"
 	./version/version.sh > $@
 
-check: ## run all linters (TODO: enable "unused", "varcheck", "ineffassign", "unconvert", "staticheck", "goimports", "structcheck")
```

Comment:
do we need to keep `check` as an alias? Or are we ok with removing it? (not sure if there's people expecting `make check` to be there)

Reply:
I dont think it's ever been used out of the GHA workflow context.

Reply:
alrighty; works for me!